input/README.md

```diff
@@ -11,9 +11,9 @@ All other columns specify additional node attributes such as prizes.
 Any nodes that are listed in a node file but are not present in one or more edges in the edge file will be removed.
 For example:
 ```
-NODEID	prize	sources	targets	active
-A	1.0		True	True
-B	3.3	True		True
+NODEID	prize	sources	targets	dummy
```

Comment:
Sorry, I wasn't clear in my last suggestion. Let's have the one example node table include both the active and dummy columns to show all supported node attributes.